HouseBuilder_Exercise2/src/houseBuilerPackage/House.java

```diff
@@ -0,0 +1,59 @@
+package houseBuilerPackage;
+
+public class House {
+	
+	private String bathroom;
+	private String bedroom;
+	private String kitchen;
+	private String livingRoom;
+	private int bathroomNum;
```

Comment:
The idea is to have a List of bathrooms and bedrooms. The number can be extracted from the List size.